installer/manifests/keptn/templates/configuration-service.yaml

```diff
@@ -48,7 +48,7 @@ spec:
       terminationGracePeriodSeconds: {{ .Values.resourceService.gracePeriod | default 60 }}
       containers:
         - name: resource-service
-          image: {{ .Values.resourceService.image.repository }}:{{ .Values.resourceService.image.tag | default .Chart.AppVersion }}
+          image: {{ include "service.image" ( dict "imageRoot" .Values.resourceService.image "global" .Values.global.keptn "defaultTag" .Chart.AppVersion) }}
```

Comment:
please also add the same snippet to the configuration-service above in this file